edx/edx-app-ios
# Swift Conversion: Splash Screen & Agreement Controller

### Description

In this PR, I have converted `OEXLoginSplashViewController` to `LoginSplashViewController` and 
`OEXUserLicenseAgreementViewController` to `AgreementWebViewController` from Objective-C to Swift.

### How to test this PR

- Verify designs of both ViewControllers. 
- Check Splash Screen Buttons are working.
- Content is properly loaded in AgreementWebViewController.
